sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -103,6 +107,75 @@
 # Helper functions =============================================================
 #
 
+
+def is_cmis_api(api):
+   return type(api) == CmisApi
```

Comment:
Wondering if we should include CCmisApi here?

Reply:
This should be done as part of a separate PR because we don't have the necessary modules to test it.